client/src/pages/donate.js

```diff
@@ -112,7 +112,7 @@ export class DonatePage extends Component {
               <div className='text-center'>
                 <PoweredByStripe />
                 <Spacer />
-                <Button onClick={this.toggleOtherOptions}>
+                <Button class='btn btn-link' onClick={this.toggleOtherOptions}>
                   {`${
                     showOtherOptions ? 'Hide' : 'Show'
                   } other ways to donate.`}
```

Comment:
This is a minor thing I noticed that's outside of the scope of this PR. Just wondering if we should remove the period from this button so it better matches the other buttons on this page:

`} other ways to donate`}`

Reply:
@scissorsneedfoodtoo although this is a button, it will look like a link.

Reply:
Sorry, should have been more clear and made the suggestion on line 118. 

What I mean is that we should probably remove the period from the text "Hide/Show other ways to donate" so it better matches the other buttons or links on the page. This seems to be the only button or link with a period on this page and in the settings.